plugins/duo_admin/komand_duo_admin/util/helpers.py

```diff
@@ -0,0 +1,23 @@
+from typing import Union
+from insightconnect_plugin_runtime.helper import return_non_empty
+
+
+def clean(item_to_clean: Union[dict, list]) -> Union[dict, list]:
+    if isinstance(item_to_clean, list):
+        return [clean(item.copy()) for item in item_to_clean]
+    if not isinstance(item_to_clean, dict):
+        return item_to_clean
+    return return_non_empty(item_to_clean.copy())
```

Comment:
also for clean function, are you able to use function from SDK? 
https://github.com/rapid7/komand-plugin-sdk-python/blob/0b2160b611924c3961a16d67d3f3ad72b05e9462/insightconnect_plugin_runtime/helper.py#L90

Reply:
We can't, because the clean method from SDK only cleans fields that have an empty string or None as their value. So instead we use the return_non_empty method from SDK to remove all empty fields in this clean method.